docs/vercel.json

```diff
@@ -559,6 +559,81 @@
       "source": "/examples/v1",
       "destination": "/docs/v1",
       "permanent": true
+    },
+    {
+      "source": "/docs/mastra-cloud/setup",
+      "destination": "/docs/deployment/mastra-cloud/setting-up",
+      "permanent": true
+    },
+    {
+      "source": "/docs/mastra-cloud/deployment",
+      "destination": "/docs/deployment/mastra-cloud/setting-up",
```

Comment:
we moved Mastra Cloud _out_ of /deployment, so I'm not sure why you're redirecting _to_ /deployment?

Reply:
These are for incorrect v0 paths generated by our version switcher.

The v1 is out of deployment! :)

1. v1: https://mastra.ai/docs/v1/mastra-cloud/deployment
2. v0 based on version switcher widget & canonical: https://mastra.ai/docs/mastra-cloud/deployment (404)
3. actual v0: https://mastra.ai/docs/deployment/mastra-cloud/setting-up